.github/workflows/ci-workload-build.yaml

```diff
@@ -0,0 +1,19 @@
+name: Build workload module
```

Comment:
It doesn't seem there's any successful run for this action:
https://github.com/GoogleCloudPlatform/cloud-spanner-samples/actions/workflows/ci-workload-build.yaml

Is this expected?

Reply:
Ah yes. Fixed the errors in the workflow. Here's a successful run - https://github.com/GoogleCloudPlatform/cloud-spanner-samples/actions/runs/1305899287